Apps/WebClient/src/ClientApp/src/components/report/ReportsComponent.vue

```diff
@@ -479,10 +479,13 @@ for (const [entryType] of reportComponentMap) {
                     </div>
                     <div>Medications:</div>
                     <MultiSelectComponent
-                        v-model="selectedMedicationOptions"
+                        :values="selectedMedicationOptions"
```

Comment:
I see you moved away from the v-model approach, was there any particular reason besides the optional props "hack" I needed to perform to have the values bind correctly?

Reply:
I ran into some problems with it, and I didn't particularly care for the name modelValue, so I went with separate props & emits for now that can be swapped to the new "v-model:" syntax once we're in Vue 3.